src/emitter.rs

```diff
@@ -127,7 +128,14 @@ pub trait Emitter {
     /// emitted.
     ///
     /// If the current token is no tag at all, this method may panic.
-    fn init_attribute(&mut self);
+    fn init_attribute_name(&mut self, reader: &R);
+
+    /// Called before the first push_attribute_value call.
+    /// If the value is wrappend in double or single quotes `quoted` is set to true, otherwise false.
+    ///
+    /// If there is no current attribute, this method may panic.
+    #[allow(unused_variables)]
+    fn init_attribute_value(&mut self, reader: &R, quoted: bool) {}
```

Comment:
this is fine but it does mean a breaking change

Reply:
Of course it's a breaking change ... adding a parameter to a trait method always is. I don't think there's anything I can do about that.